maxim/logger/components/tool_call.py

```
@@ -38,7 +39,7 @@ def get_tool_call_config_dict(
             name=config.name,
             description=config.description,
             args=config.args,
-            tags=config.tags,
+            tags=config.tags,            
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)